src/server/ServerOptions.ts

```diff
@@ -0,0 +1,15 @@
+export interface ServerOptions {
+    ServerName: string;
+    Server_IP: string;
+    Port: number;
+    MOTD: string;
+    MaxPlayers: number;
+    DefaultGamemode: number; // 0 for survival, 1 for creative
+    EnableQuery: boolean; //true for yes, false for no
+    EnableRCON: boolean; //true for yes, false for no
+    RCONPassword: string;
```

Comment:
Should be separate interface for rcon options

Reply:
k stepsis

Reply:
I'll make separate file